Dashboard/app/js/lib/views/reports/export-reports-views.js

```diff
@@ -158,6 +145,20 @@ FLOW.ExportReportsAppletView = FLOW.View.extend({
   showComprehensiveDialog: false,
   showRawDataImportApplet: false,
   showGoogleEarthButton: false,
+  reportFromDate: undefined,
+  reportToDate: undefined,
+  
+  setMinDate: function () {
+    if (this.get('reportFromDate') !== "") {
+      this.$(".to_date").datepicker("option", "minDate", this.get("reportFromDate"))
+    }
+  }.observes('this.reportFromDate'),
+
+  setMaxDate: function () {
+    if (this.get('reportToDate') !== "") { 
```

Comment:
It should be enough here to say `if (this.get('reportToDate')) {...}` same as previous `reportFromDate`